packages/circuit-ui/components/SideNavigation/components/SecondaryLinks/SecondaryLinks.tsx

```diff
@@ -78,6 +84,11 @@ function SecondaryLink({
             {label}
           </Body>
         </Skeleton>
+        {badge && (
+          <Badge variant="promo" as="span" css={badgeStyles}>
```

Comment:
Maybe in the future we can make the variant configurable, there should be a difference between a promotional badge and a notification, for example

Reply:
Yup, that's why the badge prop is an object :)